src/extensions/mountainview/MVSortingUnitView/MVSortingUnitView.tsx

```diff
@@ -0,0 +1,44 @@
+import { Grid } from "@material-ui/core"
+import React, { Fragment, FunctionComponent } from 'react'
+import { Expandable } from "../../../containers/SortingView"
+import { SortingUnitViewProps } from "../../extensionInterface"
+import sortByPriority from "../../sortByPriority"
+
+
+const MVSortingUnitView: FunctionComponent<SortingUnitViewProps> = (props) => {
```

Comment:
This is a new SortingUnitView plugin that displays all of the other sorting unit views